llvm/lib/Target/RISCV/MCTargetDesc/RISCVELFStreamer.cpp

```diff
@@ -85,6 +85,35 @@ void RISCVTargetELFStreamer::finishAttributeSection() {
                           ELF::SHT_RISCV_ATTRIBUTES, AttributeSection);
 }
 
+void RISCVTargetELFStreamer::emitNoteSection(unsigned Flags) {
```

Comment:
I know this is what AArch64 calls it, but it's a very vague name that could cover any number of note sections. Can we do better and make ours specific to what it's actually doing, i.e. something about it being a GNU properties note?